### PR TITLE
trigger DW build after merge in development or master

### PR DESCRIPTION
**Pull request by eromano**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [x] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
trigger DW build after merge in development or master

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -46,6 +46,10 @@ jobs:
     - name: Test Suite viewContent&metadata&extensions
       script: npm run build.e2e && SUITE="--suite viewer,infoDrawer,extensions" npm run e2e:docker
 
+    - stage: Trtigger DW
```
Comment (popovicsandras) [inline .travis.yml]: Typo

File: .travis.yml [hunk 2]
```diff
@@ -46,6 +46,10 @@ jobs:
     - name: Test Suite viewContent&metadata&extensions
       script: npm run build.e2e && SUITE="--suite viewer,infoDrawer,extensions" npm run e2e:docker
 
+    - stage: Trigger DW
```
Comment (adinapitul) [inline .travis.yml]: why digital workspace? this is the content-app project

File: .travis.yml [hunk 3]
```diff
@@ -46,6 +46,10 @@ jobs:
...
+      if: (branch = master OR branch = development OR (tag =~ .*beta.*)) AND type = push
+      script: ./scripts/trigger-travis.sh --pro --branch $TRAVIS_BRANCH Alfresco digital-workspace $TRAVIS_ACCESS_TOKEN
```
Comment (adinapitul) [inline .travis.yml]: same comment as above regarding naming

File: .travis.yml (hunk 2 above)
Comment (popovicsandras) [reply .travis.yml]: I think it makes sense to trigger a build of DW when its dependency (ACA) changes.
Or how is it possible to check that with any change in ACA, you haven't broken anything in DW?

Comment (eromano) [reply .travis.yml]: what @popovicsandras is saying is correct

File: .travis.yml (hunk 3 above)
Comment (eromano) [reply .travis.yml]: not sure I got but if is related to the one above what @popovicsandras is saying is correct

Comment (mauriziovitale) [reply .travis.yml]: Angular has the same but I guess they have a different pipeline...

File: .travis.yml (hunk 1 above)
Comment (eromano) [reply .travis.yml]: done

File: .travis.yml (hunk 2 above)
Comment (adinapitul) [reply .travis.yml]: do we trigger a build of all the apps when something changes in ADF ?

Comment (mauriziovitale) [reply .travis.yml]: Once we go monorepo we won't have this problem anymore :)

Comment (popovicsandras) [reply .travis.yml]: No, but ADF is versioned. We trigger builds in ADF every week, when a new beta is coming out.

Comment (eromano) [reply .travis.yml]: yes we trigger it

Comment (eromano) [reply .travis.yml]: https://github.com/Alfresco/alfresco-ng2-components/blob/development/.travis.yml#L119

Comment (popovicsandras) [reply .travis.yml]: Hmm, I was not even aware of that trigger besides the weekly package update. 🤔

Comment (adinapitul) [reply .travis.yml]: aca is not on that list https://github.com/Alfresco/alfresco-ng2-components/blob/development/scripts/travis/update/trigger-travis-children.sh#L3 :)

Comment (eromano) [reply .travis.yml]: it make sense to add it

Comment (eromano) [reply .travis.yml]: I will do it

Comment (eromano) [reply .travis.yml]: done

**Pull request merged**